README.md

```diff
@@ -216,14 +215,14 @@ And use the generated code with
 /* ... */
 
 /* The following type and function refer to the Pet type in the CDDL, which
- * has been specified as an --entry-types (-t) when invoking cddl-gen. */
+ * has been specified as an --entry-types (-t) when invoking ZCBOR. */
```

Comment:
Should we have the name of the executable upper case? I will stop highlighting upper case vs lower case at this point, since I guess my point has been made already.